tst/PeerConnectionFunctionalityTest.cpp

```diff
@@ -407,10 +407,6 @@ TEST_F(PeerConnectionFunctionalityTest, connectTwoPeersExpectFailureBecauseNoCan
 // Assert that two PeerConnections can connect and then send media until the receiver gets both audio/video
 TEST_F(PeerConnectionFunctionalityTest, exchangeMedia)
 {
-    if (!mAccessKeyIdSet) {
```

Comment:
@MushMal disabled check here, still works

Reply:
Yeah, this check needs to be removed from this test and the next one - exchangeMediaRSA